packages/web/README.md

```diff
@@ -112,14 +114,15 @@ error("still goes to stderr!");
 debug.log = console.info.bind(console);
 error("now goes to stdout via console.info");
 log("still goes to stdout, but via console.info now");
```

Comment:
Something is wrong here

Reply:
![image](https://user-images.githubusercontent.com/45826131/168019930-f544e114-a6b6-4b15-87cb-4b3296975070.png)
Could you explain more? Please ~~

Reply:
Oh my bad, thought it was a typo